internal/utils/main.go

```diff
@@ -186,8 +185,7 @@ func AddLPOperations(txMeta []xdr.OperationMeta, AssetA, AssetB xdr.Asset) []xdr
 // AddCommonFlags adds the flags common to all commands: end-ledger, stdout, and strict-export
 func AddCommonFlags(flags *pflag.FlagSet) {
 	flags.Uint32P("end-ledger", "e", 0, "The ledger sequence number for the end of the export range")
-	flags.Bool("stdout", false, "If set, the output will be printed to stdout instead of to a file")
-	flags.Bool("strict-export", false, "If set, transform errors will be reported as fatal errors instead of warnings.")
+	flags.Bool("strict-export", false, "If set, transform errors will be fatal.")
```

Comment:
Disregard `StrictExport` comment above, I see the use

Reply:
Acknowledged. Will leave it in for now, unless there's more discussion on the topic